gpu-operator/release-notes.rst

```diff
@@ -31,8 +31,119 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
    GPU Operator beta releases are documented on `GitHub <https://github.com/NVIDIA/gpu-operator/releases>`_. NVIDIA AI Enterprise builds are not posted on GitHub.
 
+
 ----
 
+.. _v24.3.0:
+
+24.3.0
+======
+
+.. _v24.3.0-new-features:
+
+New Features
+------------
+
+
+* Added support to enable NVIDIA GDRCopy v2.4.1.
+
+  When you enable support for GDRCopy, the Operator configures the GDRCopy Driver container image
+  as a sidecar container in the GPU driver pod.
+  The sidecar container compiles and installs the gdrdrv Linux kernel module.
+  This feature is supported on Ubuntu 22.04 and RHCOS operating systems and on X86_64 and ARM64 architectures.
+
+  Refer to :ref:`Chart Customization Options` for more information about the ``driver.gdrcopy`` field.
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.54.15.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit version v1.15.0
+    - NVIDIA MIG Manager version v0.7.0
+    - NVIDIA Driver Manager for K8s v0.6.8
+    - NVIDIA Kubernetes Device Plugin v0.15.0
+    - DCGM 3.3.5-1
+    - DCGM Exporter 3.3.5-3.4.1
+    - Node Feature Discovery v0.15.4
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.15.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.7
+    - NVIDIA vGPU Device Manager v0.2.6
+    - NVIDIA Kata Manager for Kubernetes v0.2.0
+
+* Added support for Kubernetes v1.29 and v1.30.
+  Refer to :ref:`Supported Operating Systems and Kubernetes Platforms`.
+
+
+* Removed support for Kubernetes pod security policy (PSP).
+  PSP was deprecated in the Kubernetes v1.21 release and removed in v1.25.
+
```

Comment:
```suggestion

* Added support for NVIDIA Network Operator v24.1.1

* Added support for the NVIDIA Orin IGX platform when configured in discrete GPU mode

```

I am not completely satisfied with how I worded the second bullet. Maybe replace "... when configured in discrete GPU mode" with "... when configured to use the discrete GPU"? @mikemckiernan I'll let you have the final say here.

Reply:
Along with the second bullet, we need to update this table in the platform support page: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/platform-support.html#supported-arm-based-platforms

We need to add a row with the following contents:
`NVIDIA Orin IGX | Ampere`

We also should edit the note box to make it clear that NVIDIA Orin IGX, a platform with an integrated GPU, is supported as long as the discrete GPU is the device being used.